pillar/edx/ansible_vars/xpro.sls

```diff
@@ -5,18 +5,21 @@
 {% set env_data = env_settings.environments[environment] %}
 {% set bucket_prefix = env_data.secret_backends.aws.bucket_prefix %}
 {% set support_email = 'support@xpro.mit.edu' %}
-{% set heroku_xpro_env_url_mapping = {
+{% set heroku_xpro_env_domain_mapping = {
     'sandbox': 'https://xpro-ci.odl.mit.edu',
-    'xpro-qa': 'https://xpro-rc.odl.mit.edu',
+    'xpro-qa': 'https://rc.xpro.mit.edu',
     'xpro-production': 'https://xpro.mit.edu'
   } %}
-{% set heroku_env = heroku_xpro_env_url_mapping['{}'.format(purpose)] %}
+{% set heroku_env = heroku_xpro_env_domain_mapping['https://{}'.format(purpose)] %}
```

Comment:
Wouldn't `purpose` get the value for the key in `heroku_xpro_env_domain_mapping` which in the default case would be xpro-qa and that would return `https://rc.xpro.mit.edu` and you'd end up with double `https`?